src/main/java/com/ss/training/utopia/agent/controller/AgentController.java

```diff
@@ -26,30 +23,15 @@
     @Autowired
 	AgentService service;
 
-    @GetMapping(path="/airport")
-    public ResponseEntity<Airport[]> getAllAirports() {
-		List<Airport> airportList = null;
-		Airport[] airportArray = null;
-		HttpStatus status = HttpStatus.OK;
-		airportList = service.readAirports();
-		if (airportList == null) // no airports exist in the database
-			status = HttpStatus.NO_CONTENT;
-		else
-        airportArray = airportList.toArray(new Airport[airportList.size()]);
-		return new ResponseEntity<Airport[]>(airportArray, status);
-	}
-
-	@PutMapping(path="/cancel")
-	public ResponseEntity<Booking> postBooking(@RequestBody Booking booking) {
+	@PostMapping(path="/booking")
```

Comment:
should be plural bookings, and maybe have utopia at the beginning of the path

Reply:
I actually don't think this one should be plural. My understanding is that it should be plural only when referring to entities in the context of a collection (such as reading a collection of entities or specifying one to update out of an existing collection).